src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -17,17 +22,11 @@ public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                new RoomNumber("21-06-40"), new Telegram("alexYeoh"), new Birthday("03/02/2000")),
```

Comment:
Just confirming we removing most sample persons?

Reply:
I was removing them for debugging purposes 😅
But as far as I see it the removal of it would not matter for now

Reply:
Is it possible to add the test cases back? As I think it will always be better to have more test coverage, or will doing so cause any issue?

Reply:
Ok I'll add it back